flaskbb/plugins/manager.py

```diff
@@ -0,0 +1,48 @@
+import pluggy
+from flaskbb.utils.helpers import parse_pkg_metadata
+
+
+class FlaskBBPluginManager(pluggy.PluginManager):
+    """Overwrites :class:`pluggy.PluginManager` to add FlaskBB
+    specific stuff.
+    """
+
+    def __init__(self, project_name, implprefix=None):
+        super(FlaskBBPluginManager, self).__init__(project_name=project_name,
+                                                   implprefix=implprefix)
+        self._plugin_metadata = {}
+        self._disabled_plugins = []
+
+    def load_setuptools_entrypoints(self, entrypoint_name):
+        """Load modules from querying the specified setuptools entrypoint name.
+        Return the number of loaded plugins. """
+        from pkg_resources import (iter_entry_points, DistributionNotFound,
+                                   VersionConflict)
+        for ep in iter_entry_points(entrypoint_name):
+            # is the plugin registered or blocked?
+            if self.get_plugin(ep.name) or self.is_blocked(ep.name):
+                self._disabled_plugins.append((ep.name, ep.dist))
```

Comment:
We should only append here if the plugin is actually disabled, not merely already loaded.

Reply:
Oops, yeah you are right. Thanks for pointing it out :)